packages/material-ui/src/Typography/Typography.test.js

```diff
@@ -159,4 +166,150 @@ describe('<Typography />', () => {
       expect(container.firstChild).not.to.have.class(classes.displayInline);
     });
   });
+
+  describe('custom theme variants', () => {
```

Comment:
Three of these tests suits can be extracted as a reusable tests, but not all of them, so I just added them here just for easier discoverability

Reply:
Considering that covering a second component should be able to scale the approach to the whole codebase, I think that it makes sense to look at the problem now. What about we move them into `describeConformance`?

Reply:
But do we even need to add tests for them?

Reply:
We have lint rule for ensuring the defaults will be present, and we have very similar tests for the hook, so on a second thought, I don't really think we need them...

Reply:
Removed the tests, the same is covered in `useThemeVariants.test.js`